src/content/en/fundamentals/performance/webpack/decrease-frontend-size.md

```diff
@@ -570,12 +603,21 @@ what this plugin does:
     })
 
 See the difference? In the plain bundle, module 0 was requiring `render` from module 1. With
-`ModuleConcatenationPlugin`, `require` is simply replaced with required function, and module 1 is
+module concatenation, `require` is simply replaced with required function, and module 1 is
 removed. The bundle has fewer modules – and less module overhead!
 
-To enable this behavior, add `ModuleConcatenationPlugin` into the list of plugins:
+To enable this behavior, **in webpack 4**, enable the `optimization.concatenateModules` option:
 
-    // webpack.config.js
+    // webpack.config.js (for webpack 4)
+    module.exports = {
+      optimization: {
+        concatenateModules: true,
```

Comment:
This is the default in webpack 4 runnign in prod mode, so this shouldn't be needed.

Reply:
Good catch, thanks! There’s actually a note about Webpack 4 in the beginning of the section, but it’s worded improperly, so I’ll update it.

In the end, the section will have a note like “If you use the production mode, just skip this section”.